on_device_training/README.md

```diff
@@ -0,0 +1,103 @@
+## Getting Started
```

Comment:
Shall we create 2 readme files? 1. This file which is under on device training can contian all the common on device training related stuff which will be applicable to other demos as well 2. Another readme file in android dir which contains everything specific to android demo. We can also plot the loss curve and get some perf metrics for the demo and add it to this readme file

Reply:
Perf (speedup) metrics are not meaningful unless we have a benchmark to compare against. I added a graph to show loss vs time. And moved directories as suggested.